packages/ui/src/scss/typography/_fonts.brutal-type.scss

```diff
@@ -0,0 +1,41 @@
+@font-face {
+  font-family: 'Brutal Type';
+  src: url('/fonts/BrutalType/brutal-type-regular.eot');
```

Comment:
Add `src: local('Brutal Type')` to all fonts for locally installed ones fallback

Reply:
commonly, we already can use only woff2, works everywhere except dead IE https://caniuse.com/#feat=woff2

Reply:
if dead IE support required, then woff2 + eot

@kazurus what's browser support we should maintain?

Reply:
@ifiri now it's ok, add only src: local('Brutal Type')